mozdownload/scraper.py

```diff
@@ -414,7 +420,11 @@ def get_build_info(self):
 
     def get_latest_build_date(self):
         """ Returns date of latest available nightly build."""
-        url = urljoin(self.base_url, 'nightly', 'latest-%s/' % self.branch)
+        if not self.application == 'mobile':
```

Comment:
I would suggest we directly use `not in ('mobile')` here so we are prepared for other applications to get added.